try:
    x = int("Pesho")
except ValueError as the_error:
    print(f"The program examined a {the_error}")
